From 3c2584db8fdde242cfbb5ea2ee29b25b2724a23f Mon Sep 17 00:00:00 2001
From: Gleb Mazovetskiy <glex.spb@gmail.com>
Date: Wed, 8 Apr 2020 19:38:21 +0100
Subject: [PATCH] Allow overriding toolchain

---
 Makefile.gcw0 | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.gcw0 b/Makefile.gcw0
index 363c3a1..534a09c 100644
--- a/Makefile.gcw0
+++ b/Makefile.gcw0
@@ -14,11 +14,11 @@ RECOMPILER = mips
 # Allows the newer toolchain to be used
 #GCC_NEW = 1
 
-RM     = rm -f
-MD     = mkdir
-CC     = /opt/gcw0-toolchain/usr/bin/mipsel-linux-gcc
-CXX    = /opt/gcw0-toolchain/usr/bin/mipsel-linux-g++
-LD     = /opt/gcw0-toolchain/usr/bin/mipsel-linux-g++
+RM     ?= rm -f
+MD     ?= mkdir
+CC     ?= /opt/gcw0-toolchain/usr/bin/mipsel-linux-gcc
+CXX    ?= /opt/gcw0-toolchain/usr/bin/mipsel-linux-g++
+LD     ?= /opt/gcw0-toolchain/usr/bin/mipsel-linux-g++
 
 SYSROOT     := $(shell $(CC) --print-sysroot)
 SDL_CONFIG  := $(SYSROOT)/usr/bin/sdl-config
-- 
2.25.1

